docs/source/index.rst

```diff
@@ -12,6 +12,8 @@ Welcome to SGCI RESOURCE INVENTORY's documentation!
    introduction
    storageDefinition
    computeDefinition
+   statusDefinition
+   OutageDefinition
```

Comment:
Should this be `outageDefinition` instead of `OutageDefinition`?

Reply:
Yes. Fixing the pull.